### DISCUSSION TITLE
Ever since switching to the new .net my crashes are non existend

### DISCUSSION THREAD

**Discussion by saskir12** [maforget/ComicRackCE#27] [General]
Just wanted ot say thanks.  Ever since modernizing parts of CR the problems I had with CR crashing after minimizing/maximising it with crashes is gone. Usually if I minimized CR to the bar, maximized it again around 3 times I got a crash with the options to Quit, Continue or Restart. Now this problem is gone.

**Comment by maforget** [comment]
Yes it was an annoying bug, it's not by chance that it was Issue #1.

The problem wasn't related to the modernity of .NET, but the way the program was continuously checking the clipboard, so it would enable the Paste buttons. The crash was actually related to windows. My theory is that the system was busy because of all the continuous check. The check is now only done when right-clicking, so in theory i could still crash, it just doesn't do it continuously. If my theory is right you would need right-click continuously with improper objects in the clipboard for it to still crash.

**Reply by saskir12** [reply]
Oh good to know. Then the one who once wrote me on Reddit that it was not the framework was right.
